src/components/Proposal/Create/SchemeForms/CreateKnownGenericSchemeProposal.tsx

```diff
@@ -61,10 +68,11 @@ class CreateKnownSchemeProposal extends React.Component<IProps, IState> {
     }
 
     const actions = props.genericSchemeInfo.actions();
+    const initialActionId = importUrlValues({ currentActionId: ""}).currentActionId;
```

Comment:
why not just call importUrlValues once for the whole state here and then you dont have to call it three times?